aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -773,6 +773,14 @@ void heaviside_kernel(TensorIterator& iter) {
   });
 }
 
+void copysign_kernel(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "copysign_cpu", [&]() {
+      cpu_kernel(iter, [](scalar_t a, scalar_t b) -> scalar_t {
+          return std::copysign(a, b);
+        });
+    });
```

Comment:
nit: the indentation here looks a bit weird to me (the ending braces don't line up with the indentation level that the starting braces happened)